lib/IRGen/GenDistributed.cpp

```diff
@@ -858,6 +858,9 @@ FunctionPointer AccessorTarget::getPointerToTarget(llvm::Value *actorSelf) {
   }
 
   auto *requirementDecl = Target.get<AbstractFunctionDecl *>();
+  if (requirementDecl->isDistributed()) {
+    requirementDecl = requirementDecl->getDistributedThunk();
```

Comment:
I don't think this is right, accessor should be already anchored on the "thunk" of the requirement, we need to figure out why it's not and fix it in GenMeta or IRGenSIL

Reply:
Good news -- this was dead code, removed.